dev/NavigationView/NavigationView.xaml

```diff
@@ -393,7 +437,7 @@
                                 OpenPaneLength="{TemplateBinding OpenPaneLength}"
                                 PaneBackground="{ThemeResource NavigationViewDefaultPaneBackground}"
                                 Grid.Row="1">
-
+                               
```

Comment:
nit: remove whitespace

Reply:
I purposefully added the extra line to be consistent with the rest of the template.

Reply:
Was referring to the white spaces in there, empty line is fine but don't think we should insert lines that contain multiple white spaces and nothing else. Empty line should be enough or do the extra white spaces help?

Reply:
So I doubled check after you point it out. I added an empty line only, no idea why Github highlighted the whitespaces?

Reply:
Huh that's weird, guess it's something only with GitHub then. Sorry about the confusion caused by this.